### DISCUSSION TITLE
Add Penpot website and GitHub repo to Brave Creators

### DISCUSSION THREAD

**Discussion by zcraber** [uxbox/uxbox#1604] [General]
Please consider adding the Penpot website and GitHub page to [Brave Creators](https://publishers.basicattentiontoken.org/) program so that users from the [Brave browser](https://brave.com/) can support the development by tipping with crypto token $BAT. It's free and super easy to set up.

**Comment by victormihalache** [comment]
Do people actually fall for that [advertisement company's](https://brave.com/brave-ads/) crypto scam?

**Reply by zcraber** [reply]
Can you explain the "crypto scam" part?

**Reply by victormihalache** [reply]
Like most cryptocurrencies, the creators made a coin, pumped it up, and are now just waiting to sell it so they make money and everyone that bought into it loses. This is even more neferious as you get BAT by seeing ads that make Brave money (masking this as "privacy friendly", even though they state they collect data, just process it on your computer, so the exploitative nature behind ads is very much there).

**Reply by mlgarchery** [reply]
I do agree the crypto world is really scammy. But the whole world is getting their data extracted and processed right now just by using Chrome (just taking an example, almost everybody does that), for free, most don't realise it.
I don't know all of what Brave does under the hood, they may be the devil.
But their browser, which is essentially Chrome based with more privacy by default and an efficient ad blocker works well (it kills me to see friends still watching ads on youtube because they don't use ads blockers !).
I personally don't like Brave Ads, which I deactivated at some points, but to get paid in BAT for agreeing to see ads seems fairer than the actual ad system to me.

**Comment by hirunatan** [comment]
Hello. Thanks for your suggestion. However, we are not considering got payed for now. We have financial muscle enough to keep it free while we develop a solid and fair business model. We are taking note of all possibilities, so this is also an useful feedback.
